src/components/academies/AcademyContentCard.jsx

```diff
@@ -109,6 +109,14 @@ const AcademyContentCard = ({
     }
     return showAllOcmCourses;
   };
+
+  const toggleButtonText = (showMoreBtnEnabled, contentType, title, contentLength) => {
+    if (showMoreBtnEnabled) {
+      return title === 'Self-paced courses' ? `< Show less ${title}` : `< Show less ${title} courses`;
```

Comment:
I think we should mark these string for translations as well.

Reply:
Make sure to run `make extract_translations` to validate it runs correctly after changes.

Reply:
@saleem-latif good catch. Thanks for reminding 👍🏻